src/enumerable.cr

```diff
@@ -644,6 +664,28 @@ module Enumerable(T)
     index { |e| e == obj }
   end
 
+  # Returns the index of the first element for which the passed block returns `true`.
```

Comment:
```suggestion
  # Returns the index of the first element for which the passed block returns truthy.
```

Reply:
Can `truthy` be used like that? Wouldn't that have to be `a truthy value` or something like that?

Reply:
Maybe just use `is truthy`? It's more concise than any of the above. And a block technically doesn't return, so it's also more precise.

Reply:
Right, it will also match the terminology of `find`.